lib/routes/callback/index.js

```diff
@@ -55,7 +58,18 @@ async function callback(server, options, done) {
       handler: async (request, response) => {
         const { id } = request.authenticate()
         const { closeContactDate, mobile, payload } = request.body
-        const { rowCount } = await server.pg.write.query(registerUpdate({ id }))
+
+        const query =
+          options.security.callbackRateLimitSeconds === undefined ||
+          options.security.callbackRateLimitSeconds === null ||
+          options.security.callbackRateLimitSeconds < 0
+            ? registerOnetimeCallback({ registrationId: id })
+            : registerRateLimitedCallback({
+                registrationId: id,
+                rateLimitSeconds: options.security.callbackRateLimitSeconds
+              })
```

Comment:
Minor simplification...

```suggestion
        const { callbackRateLimitSeconds: rateLimitSeconds } = options.security
        const query = rateLimitSeconds == null || rateLimitSeconds < 0
            ? registerOnetimeCallback({ registrationId: id })
            : registerRateLimitedCallback({ registrationId: id, rateLimitSeconds })
```

Reply:
Do you think that removing the ` === undefined` is ok?

Reply:
Yes, the `== null` is equivalent to `=== undefined || === null`

Reply:
(fwiw, there are some cases where checking them separately makes sense, such as when you're in a critical performance code path -- in which case checking for `undefined` first is faster -- but it should be more than sufficient here.